tests/test_cliCmdRegisterInstrument.py

```diff
@@ -38,7 +38,7 @@ def test_repoBasic(self):
         """Test the most basic required arguments."""
         self.run_test(["register-instrument", "here", "a.b.c"],
                       self.makeExpected(repo="here",
-                                        instrument=("a.b.c",)))
+                                        instrument=["a.b.c"]))
```

Comment:
Was this change required or you just preferred the list?

Reply:
this is because the split_commas util function was added as a callback to the `instrument_option`. `split_commas` returns a list. I wonder if it should return a tuple tho? It's a question of mutability vs immutability right?

Reply:
split_commas returning an immutable object is probably more correct given the usage scenario. You never expect someone to add more entries after they get it back.  Sometimes in the implementation code that would require a copy so we tend not to do it.

Reply:
ok, that and `textwrap.dedent` FTW